src/services/container/kubernetes/client.ts

```diff
@@ -35,7 +37,10 @@ export const getKubernetesVersion = async () => {
         compiler: string;
         platform: string;
     }>(`${currentCluster.server}/version?timeout=32s`, {
-        ...opts,
+        auth: {
+            username: kc.getCurrentUser()?.username ?? '',
+            password: kc.getCurrentUser()?.password ?? '',
+        },
```

Comment:
not sure it's fully equivalent... how about `servername`?

Reply:
Not present in axios options.
Kube command only add username, password and servername. Types are not compatible between the two.